Sources/ProcedureKit/Group.swift

```diff
@@ -52,6 +52,16 @@ open class GroupProcedure: Procedure, ProcedureQueueDelegate {
         get { return groupStateLock.withCriticalScope { _groupChildren } }
     }
 
+    /// Override of errors
+    public override var errors: [Error] {
+        get { return groupStateLock.withCriticalScope { _groupErrors.fatal } }
+        set {
+            groupStateLock.withCriticalScope {
+                _groupErrors.fatal = newValue
+            }
+        }
+    }
+
```

Comment:
@swiftlyfalling just wondering what your thoughts are on this....

Essentially this has been moved from an extension in the same file, as in Xcode 8.3 it creates a warning of "Cannot override a non-dynamic class declaration from an extension".

However, moving it into the main class definition for `GroupProcedure` causes [a failed test](https://github.com/ProcedureKit/ProcedureKit/blob/development/Tests/ProcedureKitTests/GroupTests.swift#L221):

```swift
    func test__group_cancel_with_errors_does_not_collect_errors_sent_to_children() {
        check(procedure: group) { $0.cancel(withError: TestError()) }
        XCTAssertProcedureCancelledWithErrors(group, count: 1)
    }
```

Where the test fails, as there are no group errors. Essentially, I think what is happening is that the additional error argument is not getting added to the group's fatal errors. I think, clearly, there is a timing/event issue going on.

Any thoughts?